Monika After Story/game/script-topics.rpy

```diff
@@ -1867,6 +1862,288 @@ label monika_rain_holdme:
+            show monika 5eubfu at t11 zorder MAS_MONIKA_Z with dissolve
+            m 5eubfu "Sorry, I meant {i}at least{/i} 30 seconds, ehehe..."
+            show monika 1hubfb at t11 zorder MAS_MONIKA_Z with dissolve
+        #happy
+        else:
+            m 1hubfa "Ehehe~"
+            m 3eub "Did you enjoy that?"
+            m 1hua "I sure hope so~"
+            m 1hubfb "Hugs are supposed to be good for you after all."
+
+    else:
+        #under 30 seconds
+        if mas_isMoniLove():
+            m 2ekc "Aw, are we done already?"
+            m 3eka "Could you hold me for just a bit longer?{nw}"
+            $ _history_list.pop()
+            menu:
+                m "Could you hold me for just a bit longer?{fast}"
+                "Yes.":
+                    m 1hua "Ehehe~"
+                    call monika_holdme_prep
+                    m 1hub "You're so sweet, [player]~"
+                    call monika_holdme_start
+                    call monika_holdme_reactions
+                "No.":
+                    m 2ekc "Aww..."
+                    m 2rksdlc "..."
+                    m 4eka "Please?{nw}"
+                    $ _history_list.pop()
+                    menu:
+                        m "Please?{fast}"
+                        "Yes.":
+                            m 1hub "Yay~"
+                            call monika_holdme_prep
+                            m 2ekbfb "Thanks, [player]~"
+                            call monika_holdme_start
+                            call monika_holdme_reactions
+                        "No.":
+                            m 2hksdlb "Alright, fine."
+                            m 3tsbsa "But you owe me next time, okay, [player]?"
+        elif mas_isMoniEnamored():
+            m 1ekc "Aw, is that all?"
+            m 1rksdla "I kind of wanted it to last longer than that..."
+            m 2ekbfa "Could you...{w=0.7}hold me again for a bit longer?{nw}"
+            $ _history_list.pop()
+            menu:
+                m "Could you...hold me again for a bit longer?{fast}"
+                "Yes.":
+                    m 1hubfb "Yay!"
+                    call monika_holdme_prep
+                    m 2ekbfb "Thanks, [player]~"
+                    call monika_holdme_start
+                    call monika_holdme_reactions
+                "No.":
+                    m 2ekc "Aww."
+                    m 1eka "Alright, then."
+                    m 1hub "I'll just have to wait until next time, ahaha!"
+        elif mas_isMoniAff():
+            m 1ekc "Aw, done holding me already, [player]?"
+            m 1rksdla "I was kind of hoping for it to last a little bit longer..."
+            m 1hubfa "I'm sure that won't be the last time you hold me though, so I'll look forward to next time!"
+        #happy
+        else:
+            m 1hua "That was nice~"
+            m 1hubfb "You can hold me for longer if you feel like it, you know."
```

Comment:
This line either needs to be taken out or changed, as it doesn't sound right with the line that this path leads to